modules/installation-osp-troubleshooting.adoc

```diff
@@ -11,12 +11,12 @@
 
 Unfortunately, there will always be some cases where {product-title} fails to install properly. In these events, it is helpful to understand the likely failure modes as well as how to troubleshoot the failure.
 
-This document discusses some troubleshooting options for OpenStack based
+This document discusses some troubleshooting options for RHOSP based
```

Comment:
s/RHOSP based/RHOSP-based

Reply:
This module isn't likely to see the light of day, but :+1: